ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md

```diff
@@ -4,10 +4,6 @@ Build arista.cvp.configlet configuration.
 
 Module added in version 2.9
```

Comment:
@tgodaA - the modules documentation is auto-generated using the following script 
`ansible-avd/ansible_collections/arista/avd/docs/_build/ansible2rst.py` and generated from the python code.

Let's discuss with @titom73 if we want to continue using this approach.

Reply:
Ok, @carlbuchmann let me know what changes should I make to this file.

Reply:
@carlbuchmann what is the issue here?

Reply:
@tgoda I suggest to exclude these two MD files from your changes and get the rest merged.

Reply:
Sure

Reply:
Done